<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GrabCut</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>934</width>
         <height>619</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="separator"/>
    <addaction name="prFgdAction"/>
    <addaction name="prBgdAction"/>
    <addaction name="fgdAction"/>
    <addaction name="bgdAction"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="prFgdAction"/>
   <addaction name="prBgdAction"/>
   <addaction name="separator"/>
   <addaction name="fgdAction"/>
   <addaction name="bgdAction"/>
   <addaction name="separator"/>
  </widget>
  <action name="openAction">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="fgdAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FGD</string>
   </property>
   <property name="toolTip">
    <string>foreground</string>
   </property>
  </action>
  <action name="bgdAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BGD</string>
   </property>
   <property name="toolTip">
    <string>background</string>
   </property>
  </action>
  <action name="prFgdAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PR_FGD</string>
   </property>
   <property name="toolTip">
    <string>probable foreground</string>
   </property>
  </action>
  <action name="prBgdAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PR_BGD</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
